app/utils/users.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+// @flow
+
+export function isInRole(roles, inRole) {
+    if (roles) {
+        var parts = roles.split(' ');
+        for (var i = 0; i < parts.length; i++) {
+            if (parts[i] === inRole) {
+                return true;
+            }
+        }
+    }
+
+    return false;
+}
+
+export function isGuest(user) {
+    if (user && user.roles && isInRole(user.roles, 'system_guest')) {
```

Comment:
```suggestion
    if (user && user.roles && user.roles.includes('system_guest')) {
```

Reply:
no need for the `isInRole` function, right?

Reply:
Is not exactly the same, your proposal is to check if the `system_guest` string is inside the roles, but the roles names are separated by spaces, so it can generate problems in the future, for example, I can create a role called 'system_guest2`, that would match too.